docs/blog/20240624-Daeraxa-3kStars.md

```diff
@@ -0,0 +1,136 @@
+
+We figured this might be a good chance to write a blog post going into the history of Pulsar and Atom (without which Pulsar would obviously not even exist!), covering some major events along the way.
+
+<!-- more -->
+
+<a href="https://star-history.com/#pulsar-edit/pulsar&Date"> <img src="./assets/star-history-20240624.png" height="600" /> </a>
+
+Before we go into that, we would like to take the opportunity to yet again thank our wonderful community and everyone who contributes to Pulsar. By that, we don't just mean those contributing to the codebase, but everyone: those people creating issues on GitHub, people taking part in conversations on our various social platforms, our generous financial donors, and anyone who has helped promote Pulsar by any means - upvoting, boosting toots, social media comments, and, of course, starring on GitHub.
+
+We know that the stars aren't everything, but they do give us a good idea about our general growth and community awareness. You can see from the graph that we are still on a steady upward trajectory, and we don't have plans on slowing down just yet!
+
+## History of Pulsar
+
+### Creation of Atom
+
+<img src="./assets/atom1.png" height="200" />
+
+Atom started out in 2008 being developed by GitHub's co-founder and then CEO, Chris Wanstrath, but it wasn't until 2014, with the addition of Corey Johnson and Nathan Sobo to the team, that things really got moving, and we first saw the beta being released to the public. You can view the [original announcement ](https://web.archive.org/web/20140302005920/http://blog.atom.io/2014/02/26/introducing-atom.html) but the gist is that Atom was designed from the ground up to be a no-compromise mix of usability and hackability. They had realised there was a gap in the market between the former (Sublime, Textmate) and the latter (Vim, Emacs) and wanted to fill it. Initially, the editor wasn't fully open source, but there were a bunch of libraries and packages that were. If you want to see what these early versions of Atom were like, they are still available in the original [atom/atom releases](https://github.com/atom/atom/releases). Back then, it looked a little different from what we are used to today, but you can still use that original theme in Pulsar! (*Settings* > *Themes* > *UI Theme* > Atom Dark/Light).
+
+<img src="./assets/early-atom.png" height="400" />
+
+Of course, the thing that made Atom, Atom, was its underlying framework, something developed specifically for Atom called "Atom Shell", a cross-platform framework designed to allow web technologies to be used in a desktop application. You might know it better these days as [Electron](https://www.electronjs.org/blog/electron/). So yes, if you weren't aware before, this means that the Atom editor is directly responsible for the framework that allows a whole host of popular applications to exist: Discord, VSCode, Joplin, Slack, and [far, far, more](https://www.electronjs.org/apps).
```

Comment:
```suggestion
Of course, the thing that made Atom _Atom_ was its underlying framework: something developed specifically for Atom called _Atom Shell_. It was a cross-platform framework designed to allow web technologies to be used in a desktop application. You might know it better these days as [Electron](https://www.electronjs.org/blog/electron/). So yes, if you weren't aware before, this means that the Atom editor is directly responsible for the framework that allows a whole host of popular applications to exist: Discord, VSCode, Joplin, Slack and [far, far more](https://www.electronjs.org/apps).
```

Reply:
Not sure about the move to the past tense, technically atom shell never went away, it just got renamed to electron, which is what I'm trying to portray here. Happy with the other changes but adding the "it was"  followed by "you might know it better these days" seems a little incorrect to me?

Reply:
I added the formatting but still wasn't sure about the tense change.